src/components/IssueType.jsx

```diff
@@ -34,12 +34,13 @@ const IssueType = ({
       </div>
 
       <div className={addressSearchCssClasses}>
-        <label htmlFor="location" className="address">
-          {pageFieldName}
+        <label htmlFor="location" className="address dg_label">
+          <span className="dg_label-text">{pageFieldName}</span>
         </label>
         <div className="address-input-wrapper">
```

Comment:
I had to fake this here and could not use the text input component. Since we are using a reat-autocomplete app to handle this field it didnt play nice with that library component. So instead I just molded the label to use the same format and css classes from the library instead.

Reply:
Makes sense